### PR TITLE
Pass spoken langs as url param

### PR DESCRIPTION
**Pull request by NathHorrigan**

### CODE REVIEW

**Comment by NathHorrigan** [comment]
Issue #133

File: mresponse/frontend/app/src/utils/api.js [hunk 1]
```diff
@@ -56,8 +56,21 @@ export default class Api {
     }
   }
 
-  async getReview (getNextReview = false) {
-    let response = await this.fetch(getNextReview ? `/api/review/next/` : `/api/review/`)
+  generateLanguageParam (languages) {
+    return `lang=${languages.reduce((a, b) => (a + ',' + b))}`
+  }
+
+  async getReview (languages, getNextReview = false) {
```
Comment (tm-kn) [inline mresponse/frontend/app/src/utils/api.js]: What if languages are empty? I think we need to make it optional.

File: mresponse/frontend/app/src/utils/api.js [hunk 2]
```diff
@@ -56,8 +56,21 @@ export default class Api {
     }
   }
 
-  async getReview (getNextReview = false) {
-    let response = await this.fetch(getNextReview ? `/api/review/next/` : `/api/review/`)
+  generateLanguageParam (languages) {
+    return `lang=${languages.reduce((a, b) => (a + ',' + b))}`
```
Comment (tm-kn) [inline mresponse/frontend/app/src/utils/api.js]: Isn't it easier to use `split(',')`?

File: mresponse/frontend/app/src/utils/api.js (hunk 1 above)
Comment (NathHorrigan) [reply mresponse/frontend/app/src/utils/api.js]: fixed

File: mresponse/frontend/app/src/utils/api.js (hunk 2 above)
Comment (NathHorrigan) [reply mresponse/frontend/app/src/utils/api.js]: fixed

**Pull request merged**